plugins/underflip/resorts/models/Resort.php

```diff
@@ -45,6 +46,75 @@ class Resort extends Model
         'ratings' => Rating::class,
         'numerics' => Numeric::class,
         'generics' => Generic::class,
+        'resort_images' => ResortImage::class,
+        'comments' => Comment::class,
     ];
 
+    /**
+     * The resort URL.
+     * Uses beforeSave function to eliminate duplicate error on save.
+     */
+     public function beforeSave()
+     {
+         return $this->url = sprintf('resort/%s', $this->url_segment);
+     }
+
+     /**
+      * The "total score" that represents all ratings of this resort
+      *
+      * @return Rating
+      */
+     public function getTotalScoreAttribute()
+     {
+        // Get the ratings we'll base the total score on
+        $values = $this->ratings()->pluck('value');
+
+        // Calculate the total score
+        if (is_array($values)) {
+            $score = round(array_sum($values)/count($values), 1);
+        }
+        else {
+            $score = 1;
+        }
```

Comment:
```suggestion
        $score = is_array($values)
            ? round(array_sum($values)/count($values), 1);
            : 0;
```

This does make me think... What _do_ we want as the fallback for if there's ever a resort with no ratings? Do we want its total score to be an arbitrary amount? Or do we actually want to label it "Unscored" or something? Maybe if there's no ratings, this method should just return `null`, to indicate it doesn't have a total score?

Reply:
Have addressed this in https://github.com/underflip/shredindex-backend/pull/27